jar-tool/hxformat.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Why not just use haxe-formatter's default style?

Reply:
I use a bit modified code style, but surely this setup should be removed from that repo. Just forgot about it.